src/sentry/tasks/derive_code_mappings.py

```diff
@@ -17,42 +24,114 @@
 logger = logging.getLogger("sentry.tasks.derive_code_mappings")
 
 
+@instrumented_task(
+    name="sentry.tasks.derive_code_mappings.process_organizations",
+    queue="derive_code_mappings",
+    max_retries=0,
+)  # type: ignore
+def derive_missing_codemappings(dry_run=False) -> None:
+    """
+    Queue tasks to process each organization to derive missing codemappings.
+    """
+    organizations = Organization.objects.filter(status=OrganizationStatus.ACTIVE)
+    for _, organization in enumerate(
+        RangeQuerySetWrapper(organizations, step=1000, result_value_getter=lambda item: item.id)
+    ):
+        if not features.has("organizations:derive-code-mappings", organization):
+            continue
+
+        # Create a celery task per organization
+        project_stacktrace_paths = identify_stacktrace_paths.delay(organization.id, dry_run=dry_run)
+        if not project_stacktrace_paths:
+            continue
+
+        integration = None
+        try:
+            integration = Integration.objects.filter(
+                organizations=organization.id,
+                provider="github",
+            )
+        except Integration.DoesNotExist:
+            logger.exception(f"Github integration not found for {organization.id}")
+
+        if integration is None or not integration.exists():
+            continue
+
+        integration = integration.first()
+        organization_integration = OrganizationIntegration.objects.filter(
+            organization=organization, integration=integration
+        )
+        if not organization_integration.exists():
+            continue
+
+        organization_integration = organization_integration.first()
+        install = integration.get_installation(organization.id)
+        trees: JSONData = install.get_trees_for_org()
+        for project, stacktrace_paths in project_stacktrace_paths.items():
+            code_mappings = derive_code_mappings(stacktrace_paths, trees)
+            set_project_codemappings(code_mappings, organization, organization_integration, project)
+
+
+# Given a list of code mappings, create a new repository project path
+# config for each mapping.
+def set_project_codemappings(
+    code_mappings: List[CodeMapping],
+    organization: Organization,
+    organization_integration: OrganizationIntegration,
+    project: Project,
+) -> None:
+    for code_mapping in code_mappings:
+        serializer = RepositoryProjectPathConfigSerializer(
+            data={
+                "project_id": project.id,
+                "stack_root": code_mapping.stacktrace_root,
+                "source_root": code_mapping.source_path,
+                "repo_id": code_mapping.repo,
+            },
+            context={
+                "organization": organization,
+                "organization_integration": organization_integration,
+            },
+        )
+        if serializer.is_valid():
```

Comment:
This won't validate the data currently - the `default branch` is required and not available in CodeMapping. @armenzg how do we want to handle this?